Code/MinimalLib/docker/Dockerfile

```diff
@@ -65,3 +66,9 @@ RUN make -j2 RDKit_minimal && \
 # run the tests
 WORKDIR /src/rdkit/Code/MinimalLib/tests
 RUN nodejs tests.js
+
+# Copy js and wasm rdkit files to use in browser
+# This feature requires the BuildKit backend
+# https://docs.docker.com/engine/reference/commandline/build/#custom-build-outputs
+FROM scratch as export-stage
+COPY --from=build-stage /src/rdkit/Code/MinimalLib/demo /
```

Comment:
Is no newline at end of file deliberate?  It always looks a bit untidy to me, unless it's necessary.

Reply:
new lines added everywhere :+1: